adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -108,6 +110,7 @@ private AuthenticationConstants() { }
          * Waiting for broker package to be installed, and resume request in broker. 
          */
         public static final int BROKER_REQUEST_RESUME = 2006;
+		public static final int BROWSER_CODE_DEVICE_REGISTER = 2006;
```

Comment:
It seems public ADAL and Unity gave different names for the same code value. BROWSER_CODE_DEVICE_REGISTER is used by ad-accounts pkg, since we should not touch it at the same time, the best solution is to keep both at same time for now. After the merging is done, then we could go back and decide which one we should delete.